inference-engine/tests/ie_test_utils/functional_test_utils/src/layer_test_utils/summary.cpp

```diff
@@ -68,6 +68,16 @@ void Summary::updateOPsStats(const ngraph::NodeTypeInfo &op, const PassRate::Sta
     }
 }
 
+std::string Summary::getOpVersion(const ngraph::NodeTypeInfo &type_info) {
+    for (size_t i = 1; i <= opsets.size(); i++) {
+        const auto type_infos = opsets[i].get_type_info_set();
```

Comment:
Will be better to add the `&` in order to get the value by reference.

But the best variant is to use the opset API. For example: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/include/ngraph/opsets/opset.hpp#L72